Test/FileStorage/ControllerFunctionsSpec.hs

```diff
@@ -0,0 +1,70 @@
+module Test.FileStorage.ControllerFunctionsSpec where
+
+import Test.Hspec
+import IHP.Prelude
+import IHP.FileStorage.ControllerFunctions
+import IHP.Controller.Context
+import IHP.FrameworkConfig
+import Network.Wai as Wai (defaultRequest)
+import Network.Wai.Parse (FileInfo(..))
+import IHP.Controller.RequestContext
+import IHP.FileStorage.Types
+import IHP.FileStorage.Config
+
+tests :: Spec
+tests = describe "IHP.FileStorage.ControllerFunctions" $ do
+
+    let config :: ConfigBuilder
+        config = do
+            initStaticDirStorage
+
+    let withFrameworkConfig = IHP.FrameworkConfig.withFrameworkConfig config
+
+    describe "storeFileWithOptions" $ do
+        it "returns the objectPath without the baseUrl" $ do
+            withFrameworkConfig \frameworkConfig -> do
+                context <- createControllerContext frameworkConfig
+                let ?context = context
+
+                let fileInfo = FileInfo
+                        { fileName = "test.txt"
+                        , fileContentType = "text/plain"
+                        , fileContent = "Hello, world!"
+                        }
+
+                -- We pass the UUID that will be used as the filename, so we can easily assert the objectPath.
+                let options :: StoreFileOptions = def
+                        { fileName = Just "4c55dac2-e411-45ac-aa10-b957b01221df"
+                        , directory = "Test.FileStorage.ControllerFunctionsSpec"
+                        }
+
+                result <- storeFileWithOptions fileInfo options
+
+                result.url `shouldBe` ("Test.FileStorage.ControllerFunctionsSpec/4c55dac2-e411-45ac-aa10-b957b01221df")
```

Comment:
Wouldn't it be better to prepend a `/` to the path? Otherwise you cannot really use the url e.g. in an `<img>` tag or for something else.

```suggestion
                result.url `shouldBe` ("/Test.FileStorage.ControllerFunctionsSpec/4c55dac2-e411-45ac-aa10-b957b01221df")
```

Reply:
Right, I thought I've done it, maybe I've reverted. Let me check :smile:

Reply:
Fixed in https://github.com/digitallyinduced/ihp/pull/1990/commits/adfaf3c28ae352df0e7cbd9ae3e8e65e7279dbd4